python/paddle/fluid/tests/unittests/test_fill_constant_batch_size_like_op.py

```diff
@@ -21,6 +21,7 @@
 
 class TestFillConstantBatchSizeLikeWhenFirstDimIsBatchSize(OpTest):
     def setUp(self):
+        self.test_gc = True
```

Comment:
Which unit tests should use `self.test_gc = True`?

Reply:
Currently, only op that registers `NoNeedBufferVarsInference` should add this flag.